src/main/target/OMNIBUSF4V6/target.h

```diff
@@ -0,0 +1,191 @@
+#define GYRO_MPU6500_ALIGN      CW180_DEG
+#define ACC_MPU6500_ALIGN       CW180_DEG
+
+// OmnibusF4 Nano v6 has a MPU6000
+#define USE_GYRO_MPU6000
+#define USE_ACC_MPU6000
+#define MPU6000_CS_PIN          PA4
+#define MPU6000_SPI_BUS         BUS_SPI1
+#define GYRO_MPU6000_ALIGN      CW180_DEG
+#define ACC_MPU6000_ALIGN       CW180_DEG
+
+#define USE_MAG
+#define MAG_I2C_BUS             BUS_I2C1
+#define USE_MAG_HMC5883
+#define USE_MAG_QMC5883
+#define USE_MAG_IST8310
+#define USE_MAG_MAG3110
+#define USE_MAG_LIS3MDL
+
+#define USE_BARO
+
+#define USE_BARO_BMP280
+#define BMP280_SPI_BUS        BUS_SPI3
+#define BMP280_CS_PIN         PB3
+
+#define USE_PITOT_MS4525
+#define PITOT_I2C_BUS           BUS_I2C1
+
+#define USE_RANGEFINDER
+#define RANGEFINDER_I2C_BUS     BUS_I2C1
+#define USE_RANGEFINDER_HCSR04_I2C
+
+#define USE_VCP
+#define VBUS_SENSING_PIN        PC5
+#define VBUS_SENSING_ENABLED
+
+#define USE_UART_INVERTER
+
+#define USE_UART1
+#define UART1_RX_PIN            PA10
+#define UART1_TX_PIN            PA9
+
+
+
+#define USE_UART2
+#define UART2_RX_PIN            NONE
+#define UART2_TX_PIN            PA2
+
+#define USE_UART3
+#define UART3_RX_PIN            PB11 
+#define UART3_TX_PIN            PB10 
+
+#define USE_UART4
+#define UART4_RX_PIN            PA1
+#define UART4_TX_PIN            NONE
+
+#define USE_UART6
+#define UART6_RX_PIN            PC7
+#define UART6_TX_PIN            PC6
+
+#define USE_SOFTSERIAL1
+#define SOFTSERIAL_1_RX_PIN     NONE
+#define SOFTSERIAL_1_TX_PIN     NONE     // Clash with UART1_TX, needed for S.Port
+
+#define SERIAL_PORT_COUNT       7       // VCP, USART1, USART2, USART3, USART4, USART6, SOFTSERIAL1
+
+#define USE_SPI
+
+#define USE_SPI_DEVICE_1
+#define SPI1_NSS_PIN            PA4
+#define SPI1_SCK_PIN            PA5
+#define SPI1_MISO_PIN           PA6
+#define SPI1_MOSI_PIN           PA7
+
+#define USE_SPI_DEVICE_2
+#define SPI2_NSS_PIN            PB12
+#define SPI2_SCK_PIN            PB13
+#define SPI2_MISO_PIN           PB14
+#define SPI2_MOSI_PIN           PB15
+
+#define USE_SPI_DEVICE_3
+#define SPI3_NSS_PIN            PA15
+#define SPI3_SCK_PIN            PC10
+#define SPI3_MISO_PIN           PC11
+#define SPI3_MOSI_PIN           PC12
+
+#define USE_OSD
+#define USE_MAX7456
+#define MAX7456_SPI_BUS         BUS_SPI3
+#define MAX7456_CS_PIN          PA15
+
+#define ENABLE_BLACKBOX_LOGGING_ON_SPIFLASH_BY_DEFAULT
+#define M25P16_CS_PIN           PB12
+#define M25P16_SPI_BUS          BUS_SPI2
+#define USE_FLASHFS
+#define USE_FLASH_M25P16
+
+#define USE_ADC
+#define ADC_CHANNEL_1_PIN               PC1
+#define ADC_CHANNEL_2_PIN               PC2
+#define ADC_CHANNEL_3_PIN               PA0
+
+#define CURRENT_METER_ADC_CHANNEL       ADC_CHN_1
+#define VBAT_ADC_CHANNEL                ADC_CHN_2
+#define RSSI_ADC_CHANNEL                ADC_CHN_3
+
+#define SENSORS_SET (SENSOR_ACC | SENSOR_BARO)
+
+#define USE_LED_STRIP
+#define WS2811_PIN                      PB6
```

Comment:
I think `PB6` is used for i2c when `USE_I2C_DEVICE_1` is defined. Does this matter?

Reply:
PB8 and PB9 are used for i2c1 for this board.

Reply:
OK great I missed that `USE_I2C_PULLUP` is defined in `common.h`. This might help me with https://github.com/iNavFlight/inav/issues/4025

Reply:
Hi Alex, I'll have a look at your issue. Did you verify (with a multimeter) which pins the jst points to? note that ff its pb6/7, then uart 1 must be disabled. snakkes, Andrew